discussion_agents/cog/agent/react.py

```diff
@@ -134,7 +156,7 @@ def generate(self, question: str, reset: bool = True) -> str:
                         "Could not find that page, please try again."
                     )
 
-            elif action_type.lower() == "lookup":
+            elif action_type.lower() == "lookup" & self.type_benchmark == "HotpotQA":
```

Comment:
It looks like there's a slight functionality change between hotpotqa and fever. We should avoid having explicit checks for benchmark_type that affect functionality (if possible). Is it possible to maybe instead of having an explicit check for hotpotqa here, you can add a new attribute to this class like "use_lookup: bool".

```suggestion
            elif action_type.lower() == "lookup" & self.type_benchmark == "hotpotqa":
```